docs/getting-started-guides/kubectl.md

```diff
@@ -0,0 +1,63 @@
+---
+---
+
+<style>
+li>.highlighter-rouge {position:relative; top:3px;}
+</style>
+
+## Overview
+
+kubectl is the command line tool you will use when interacting with kubernetes.
```

Comment:
Stolen from other docs.

`kubectl` is a command line interface to help you run commands against Kubernetes clusters.